ace/sources/Springer.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Springer",
   "identifiers": [
-    "- Springer</title>",
-    "SpringerLink</title>"
```

Comment:
Are these identifiers no longer relevant since you are combining/modifying the HTML for springer articles? Are there any cases where you do not have to combine multiple HTML files?

Reply:
You are correct, I need to have the -Springer and SpringerLink</title> instead.